outside.go

```diff
@@ -255,18 +255,17 @@ func (f *Interface) handleHostRoaming(hostinfo *HostInfo, udpAddr netip.AddrPort
 }
 
 func (f *Interface) handleEncrypted(ci *ConnectionState, addr netip.AddrPort, h *header.H) bool {
-	// If connectionstate exists and the replay protector allows, process packet
-	// Else, send recv errors for 300 seconds after a restart to allow fast reconnection.
-	if ci == nil || !ci.window.Check(f.l, h.MessageCounter) {
-		if addr.IsValid() {
-			f.maybeSendRecvError(addr, h.RemoteIndex)
-			return false
-		} else {
-			return false
-		}
+	// If connectionstate exists, process packet
+	// Else, send recv error to allow fast reconnection.
+	if ci != nil {
```

Comment:
We still want to avoid handling the encrypted packet if the window check fails, just not bother sending the recv_error. 

There is also the accompanying bug in hostmap.go that #1459 speaks to

Reply:
> We still want to avoid handling the encrypted packet if the window check fails, just not bother sending the recv_error.

oh crap, good catch, thanks

> There is also the accompanying bug in hostmap.go that https://github.com/slackhq/nebula/pull/1459 speaks to

do we still actually want this? Wanting 4 packets hurts the fast-reconnect story

Reply:
Well the current behavior increments a counter and ignores it so I'd just remove the function entirely. Honestly would take a big hammer to a lot of this.